packages/rest/README.md

```diff
@@ -52,16 +52,36 @@ Application options.
 const app = new RestApplication({
   rest: {
     port: 3001,
+
+    // Template the OpenAPI spec, which will be filled with `paths` from
+    // controllers.
+    openApiSpec: {
+      info: {
+        title: 'My Application',
+        version: '1.0.0',
+      },
+      servers: [{url: 'http://127.0.0.1:8080'}],
+    },
+
+    // Optional configuration for API explorer and spec
+    apiExplorer: {
+      // URL for the hosted API Explorer UI
+      // default to https://loopback.io/api-explorer
+      explorerUrl: 'http://petstore.swagger.io',
+      // Set `servers` based on HTTP request headers, default to `false`
+      setServersFromRequest: false,
```

Comment:
I don't think this is a `apiExplorer` related property either but now sure what a better place for this might be.

Reply:
How about moving this property from `rest.apiExplorer.setServersFromRequest` to something like `rest.updateOpenApiServersFromRequest`?

Reply:
Uh, `rest.updateOpenApiServersFromRequest` is awfully long. How about `rest.setServerUrlsFromRequest`?

Reply:
Or how about just `rest.serverUrlsFromRequest`?